introduction.md

```diff
@@ -0,0 +1 @@
+Dave is taking code school courses to see how he likes them.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.